### PR TITLE
6 sept/rspec

### PR DESCRIPTION
**Pull request by ShahirMansuri786**

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-    belongs_to :user
-    has_many_attached :post_images , dependent: :destroy
-    has_many :likes , as: :likeable , dependent: :destroy
-    has_many :comments , dependent: :destroy
-    validates :caption , presence: true
+	belongs_to :user
+	has_many_attached :post_images , dependent: :destroy
+	has_many :likes , as: :likeable , dependent: :destroy
+	has_many :comments , dependent: :destroy
+	validates :caption , presence: true
```
Comment (Sahilchouhan20) [inline app/models/post.rb:6]: you should add a blank line at end of the code.

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  describe "Write test cases for comment" do
+    context "Validation for comment" do
+      it { should validate_presence_of(:comment_text) }
+    end
+    context "Associatio for comment" do
+      it { should have_many(:likes) }
+      it { should belong_to(:post)}
+    end
+  end
+end
```
Comment (Sahilchouhan20) [inline spec/models/comment_spec.rb:13]: you should add a blank line at end of the code.

**Review by Sahilchouhan20 — COMMENTED** [review]
you should add a blank line at end of the code.

File: app/models/post.rb (hunk 1 above)
Comment (ShahirMansuri786) [reply app/models/post.rb:6]: ok sure I will remove  it

File: spec/models/comment_spec.rb (hunk 2 above)
Comment (ShahirMansuri786) [reply spec/models/comment_spec.rb:13]: ok

**Pull request merged**